### PR TITLE
[W4.11e][T09-1]Chen Shaowei

### PR DESCRIPTION
**Pull request by tuesmiddt**

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -9,7 +9,15 @@
 import org.junit.Test;
 
 public class UtilsTest {
-
+    
+    @Test
+    public void isAnyNull() {
+        assertFalse(Utils.isAnyNull());
+        assertTrue(Utils.isAnyNull((Object) null));
```
Comment (novellius) [inline test/java/seedu/addressbook/common/UtilsTest.java:16]: Perhaps you can use comments to separate different test cases for better readability? For example,
// Empty list
assertFalse(Utils.isAnyNull());

// Non-empty list with nulls
assertTrue(...);
...

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -9,7 +9,15 @@
...
+        assertTrue(Utils.isAnyNull(null, 1));
+        assertTrue(Utils.isAnyNull(new Object[]{null, null, 1}));
+        assertTrue(Utils.isAnyNull(1, 2, 3, null));
+    }
```
Comment (novellius) [inline test/java/seedu/addressbook/common/UtilsTest.java:20]: How about adding test cases for non-empty lists with no nulls?

**Review by novellius — COMMENTED** [review]
Good work @tuesmiddt . See my comments for suggestions. Please close this PR after reading the comments.
